cardano-cli/src/Cardano/CLI/Ops.hs

```diff
@@ -354,7 +360,11 @@ instance Show CliError where
     = "Error submitting update proposal: " <> (T.unpack $ renderRealPBFTError err)
   show (VerificationKeyDeserialisationFailed fp err)
     = "Verification key '" <> fp <> "' read failure: "<> T.unpack err
-  show (VRFCliError err) = show $ renderVRFError err
+  show (VRFCliError err) = T.unpack $ renderVRFError err
+  show (IOError fp ioe)
```

Comment:
`IOError` & `AesonDecode` comes from `Cardano.CLI.Shelley.Run.Genesis`. Would it make sense to group the errors with e.g `ShelleyGenesisCliError` and a rendering function, similar to other modules e.g. `AddressCliError`?

Reply:
Six of one and half-a-dozen of the other.